specification/datacollaboration/resource-manager/Microsoft.DataCollaboration/preview/2020-05-04-preview/DataCollaboration.json

```diff
@@ -4274,6 +4631,10 @@
             "name": "kind",
             "modelAsString": true
           }
+        },
+        "systemData": {
+          "$ref": "#/definitions/SystemData",
```

Comment:
Suggest to ref the definition from types.json under common-type

Reply:
You mean something like this:
`"$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/systemData"` ?

I see how it will work in this repo, but we use this swagger file to auto generate sdks as part of our repo build process and it will need to be able to resolve the ref. I wouldn't want to copy the common files to our reop - can I reference the original one somehow?

Reply:
@akning-ms

Reply:
And in any case If this is not a hard blocker I'd like to make that in a separate change.

Reply:
ok, make sense, will ping you for some detail offline for your build process to autogen sdks as our pipeline alread integrate SDK generation